webapp-django/crashstats/crashstats/templates/crashstats/topcrasher.html

```diff
@@ -33,6 +33,9 @@
                   {% endif %}
                 {% endif %}
             </ul>
+            <a href="{{ url('topcrashers:topcrashers') }}?product={{ product }}&amp;version={{ versions[0] }}&amp;date_range_type={{ date_range_type }}&amp;process_type={{ crash_type.replace('all', 'any') }}&amp;platform={{ os_name }}&amp;_facets_size={{ result_count }}&amp;days={{ days }}">
+                See the new Top Crashers report
```

Comment:
That's the exact same phrasing I have been using for report/list/ and signature report. Should I also change it there?